app/src/main/res/layout/activity_main.xml

```diff
@@ -11,22 +11,33 @@
         android:id="@+id/button_main_activity"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/main_activity_count_text_margin_top"
         android:backgroundTint="@color/cardview_dark_background"
         android:text="@string/text_button_main_activity"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toTopOf="@id/text_view_main_activity"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintStart_toStartOf="parent" />
 
     <TextView
         android:id="@+id/text_view_main_activity"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/main_activity_count_text_margin_top"
         android:text="@string/text_view_main_activity"
```

Comment:
please use a representative name also, like title_main_activity or something like that

Reply:
done